app/operations/candidates/add.rb

```diff
@@ -38,14 +24,30 @@ class Candidates::Add < ApplicationOperation
         type: Types::Strict::String
       ).optional
     ),
+    file_id_to_change_cv_status?: Types::Strict::String,
```

Comment:
Do we need this property? The same for file_id_to_remove/remove_avatar?

Reply:
You are right.  I don't see these arguments being passed to the operation and used in it.

Reply:
Fixed in [58ab709](https://github.com/freeats/freeats/pull/103/commits/58ab70999ea21bb6ff9e03405311ebf6f4818e4d)